index.html

```diff
@@ -260,25 +288,27 @@ <h2>Contact me</h2>
         </p>
       </div>
 
-      <input
-        type="text"
-        name="name"
-        required
-        maxlength="30"
-        placeholder="Name"
-      />
+      <form action="https://formspree.io/f/xpzbgplp" method="POST" class="form">
+        <input
+          type="text"
+          name="name"
+          required
+          maxlength="30"
+          placeholder="Name"
+        />
 
-      <input type="email" name="email" required placeholder="Email" />
+        <input type="email" name="email" required placeholder="Email" />
 
-      <textarea
-        class="text-area"
-        name="message"
-        maxlength="500"
-        required
-        placeholder="write your message here"
-      ></textarea>
+        <textarea
+          class="text-area"
+          name="message"
+          maxlength="500"
+          required
+          placeholder="write your message here"
+        ></textarea>
 
-      <button class="form-btn" type="submit">Get in Touch</button>
-    </form>
+        <button class="form-btn" type="submit">Get in Touch</button>
```

Comment:
- [ ] Kindly edit the color for the hover to be the same as the previous buttons to match the Figma design🔥🔥

Reply:
Correction effected.
Thanks for the review.